modules/oadp-1-3-backing-csi-snapshots.adoc

```diff
@@ -29,11 +29,11 @@ metadata:
   namespace: openshift-adp
 spec:
   csiSnapshotTimeout: 10m0s
-  defaultVolumesToFsBackup: false
+  defaultVolumesToFsBackup: true
```

Comment:
there is no need to pass this flags on DPA if you set the defaultVolumesToFsBackup flag in the DPA CR. All the consecutive backups will follow the same procedure. 

For example:- if you created a dpa with defaultSnapshotMoveData flag set to true then you don't have to specify this snapshotMoveData flag in backupCR as it will be automatically added.

Reply:
@PrasadJoshi12 
So I should remove the `true` from `defaultVolumesToFsBackup` and `defaultSnapshotMoveData` and move the call-out to the DPA CR?

Reply:
Also, as noted above, defaultVolumesTofsBackup should be true if most volumes will use fs-backup instead of data mover (per-volume opt-out to use data mover), and it should be false if most volumes will use data mover (per-volume opt-in to use fs-backup).